arch/arm64/src/imx9/imx9_flexcan.c

```diff
@@ -1645,6 +1656,35 @@ static int imx9_ioctl(struct net_driver_s *dev, int cmd,
         }
         break;
 #endif
+
+#ifdef CONFIG_NETDEV_CAN_FILTER_IOCTL
+      case SIOCACANSTDFILTER: /* Set STD ID CAN filter */
+        {
+          struct imx9_driver_s *priv = (struct imx9_driver_s *)dev;
+          struct can_ioctl_filter_s *req =
+            (struct can_ioctl_filter_s *)((uintptr_t)arg);
+          ret = imx9_add_filter(priv, req->ftype, 0, req->fid1, req->fid2);
```

Comment:
maybe you should add a DEBUGASSERT() before using req, to guarantee it is not NULL